packages/@storefront/flux-capacitor/test/unit/core/requests/utils.ts

```diff
@@ -479,6 +479,38 @@ suite('requests helpers', ({ expect, stub, spy }) => {
     });
   });
 
+  describe('productDetails()', () => {
+    it('should create a product details request using history', () => {
+      const state: any = { area: 'oof', collection: 'rab', };
+      const request = { area: 'foo', collection: 'bar', };
+      const history = { request: { area: 'foo', collection: 'bar', } };
+      const overrides = request;
+
```

Comment:
Since the 'act' and 'assert' steps have been collapsed into on, everything above the `expect()` invocation is technically part of the 'arrange' step. Lets remove the new lines between the `const` declaration and the `stub`, as well as between the `stub` function calls.